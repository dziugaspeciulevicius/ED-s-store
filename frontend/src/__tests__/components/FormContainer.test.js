
import React from "react";
import { shallow } from "enzyme";
import "../../setupTests";
import FormContainer from "../../components/FormContainer.component";

describe("Form container tests", () => {
    it("Renders <FormContainer /> component correctly", () => {
      // shallow creates instance of a component
      expect(shallow(<FormContainer />)).toMatchSnapshot();
    });
});
