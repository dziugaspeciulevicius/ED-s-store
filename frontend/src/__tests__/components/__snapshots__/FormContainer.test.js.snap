// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form container tests Renders <FormContainer /> component correctly 1`] = `ShallowWrapper {}`;
